scripts.js

```diff
@@ -0,0 +1,18 @@
+function displayButtonText(n) {
+
+    var buttonId = document.getElementById('button');
+    var buttonClass = buttonId.getElementsByClassName('button');
+    console.log(buttonClass);   
+
+    var buttonCount = buttonClass.length;
+    console.log('Liczba przyciskow z id button i klasa button:', buttonCount);
+
+    var n = buttonCount;
```

Comment:
po co to? `buttonCount` jest już zmienną, po co tworzyć `n`, która przechowa to samo?